config/manager/manager.yaml

```diff
@@ -50,7 +50,7 @@ stringData:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: user-ca-bundle
+  name: pc-trusted-ca-bundle
```

Comment:
Any concerns with backwards compatibility changing the CM name here?

Reply:
Will do more testing with clusterctl upgrade

Reply:
tested following steps manually and found working
<pre>
clusterctl init -i nutanix => installs v1.2.4
clusterctl upgrade plan => shows up to date
# add local provider path in ~/.cluster-api/clusterctl.yaml and then proceed
clusterctl upgrade plan => shows that newer version v1.3.99 is available
clusterctl upgrade apply -i nutanix:v1.3.99
clusterctl upgrade plan => shows up to date
make test-create-cluster => found that cluster got created
</pre>